packages/openneuro-components/src/search-page/FiltersBlock.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Button } from '../button/Button'
+import './filters-block.scss'
+
+export interface FiltersBlockProps {}
```

Comment:
Assuming this will be filled in with things like `modality_selected`, `ageRange`, etc?